connection/connection_params.ts

```diff
@@ -107,7 +114,24 @@ export type ClientControls = {
    * - `strict` : deno-postgres parses the data into JS objects, and if a parser is not implemented, it throws an error
    * - `raw` : the data is returned as Uint8Array
    */
-  decode_strategy?: "string" | "auto";
+  decode_strategy?: DecodeStrategy;
+
+  /**
+   * A dictionary of functions used to decode (parse) column field values from string to a custom type. These functions will
+   * take presedence over the `decode_strategy`. Each key in the dictionary is the column OID type number and the value is
+   * the decoder function. You can use the `Oid` object to set the decoder functions.
+   *
+   * @example
+   * {
+   *   //   16 = Oid.bool : convert all boolean values to numbers
+   *   '16': (value: string) => value === 't' ? 1 : 0,
+   *   // 1082 = Oid.date : convert all dates to dayjs objects
+   *   [1082]: (value: string) => dayjs(value),
+   *   //   23 = Oid.int4 : convert all integers to positive numbers
+   *   [Oid.int4]: (value: string) => Math.max(0, parseInt(value || '0', 10)),
+   * }
```

Comment:
```suggestion
   * ```jsonc
   *  {
   *   //   16 = Oid.bool : convert all boolean values to numbers
   *   '16': (value: string) => value === 't' ? 1 : 0,
   *   // 1082 = Oid.date : convert all dates to dayjs objects
   *   [1082]: (value: string) => dayjs(value),
   *   //   23 = Oid.int4 : convert all integers to positive numbers
   *   [Oid.int4]: (value: string) => Math.max(0, parseInt(value || '0', 10)),
   * }
   * ```
```
So the snippet is correctly rendered in docs.

Reply:
Does `jsonc` accept js logic? Not sure if its rendering correctly now 
<img width="957" alt="image" src="https://github.com/denodrivers/postgres/assets/44002321/7e4f092c-95db-44f8-9285-3d1670dfcea5">

Reply:
Oh, it should actually be `ts`. Sorry.